Sanya_form.HTML

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+   
+    <form>
+      <b><label>First Name:</label></b>
+      <input 
+            id="first_name" 
+            type="text" 
+            placeholder="Name"
+            maxlength="10" 
+            minlength="10"/>
+        
+        <br/>
+        
+      <b><label>Last Name:</label></b>
+      <input id="last_name" type="text" placeholder="Name" />
+        
+        <br/>
+
+      <b><label>Date_of_Birth:</label></b>
+      <input type="date" />
+
+        <br/>
+        
+      <b><label>Email:</label></b>
+      <input type="email" placeholder="email"/>
+        
+        <br/>
+        
+      <b><label>Gender:</label></b>
+
+        <br/>
+
+      <b><label>Mobile:</label></b>
+      <input type="mobile" />
+
+        <br/>
+
+      <b><label>Address:</label></b>
+      <input type="address" maxlength="20" />
+
+        <br/>
+
+      <b><label>Male:</label></b>
+      <input id="male" type="radio" />
+
+        <br/>
+
+      <b><label>Female:</label></b>
+      <input id="female" type="radio" name="gender">
+
+        <br/>
+
+      <b><label>Other:</label></b>
+      <input id="other" type="radio" name="gender">
+        
+        <br/></b></b>
+      
+      <input type="submit" type="color-green">
```

Comment:
You can only have one "type" attribute. I think you are looking for the "style" attribute